src/components/SignOutButton/SignOutButton.jsx

```diff
@@ -0,0 +1,49 @@
+import {useEasyauth} from '../../hooks/useEasyauth/useEasyauth.jsx';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export const SignOutButton = ({callback, children}) => {
+  const auth = useEasyauth();
+  if (!children) {
+    return (
+      <a
+        href={
+          process.env.REACT_APP_EASYAUTH_APP_URL +
+          '/logout?target=' +
+          btoa(callback || window.location.href)
+        }
+      >
+        <button
+          onClick={() => {
+            auth.removeUser();
```

Comment:
Try this: Configure removeUser such that after removing user it redirects to the logout URL.

Or.

Prepare a special page for logout, the page will only call removeUser.
In the logout button, just link to the logout url with redirect target to the special logout page.

Reply:
@ghoshbishakh  , need  end_session_endpoint from .well-known/openid-configuration

Reply:
resolved in commit 2f7699f2d46f4a589f1b2438dfaf2866a8dd7c72.